### ISSUE TITLE
add timestamp of last update

### ISSUE THREAD

**Issue by lydiapintscher** [frimelle/tree-of-life#4]
The tree of life should have a timestamp of the last update so people can easily see how up-to-date it is. The footer seems like a good place for it.

**Comment by mariushoch** [comment]
There is a " Date of last update: ..." in the footer. So I guess this is fixed?

**Comment by frimelle** [comment]
Not really. It's just a hard coded date. Wanted to keep that open to remind me to add it dynamically for the times I will have a cron job.
